tests/dispatch/tests.py

```diff
@@ -96,6 +96,15 @@ def test_garbage_collected(self):
         self.assertEqual(result, [])
         self.assertTestIsClean(a_signal)
 
+    def test_send_robust_logger_exception(self):
+        def fails(val, **kwargs):
+            raise ValueError()
+        b_signal.connect(fails)
+        with self.assertLogs("django.dispatch.dispatcher"):
```

Comment:
maybe I'm being a stickler, but I'd make extra assertions on the log record's level and message

Reply:
Yes, we should check if it's on `ERROR` level:
```python
with self.assertLogs('django.dispatch.dispatcher', 'ERROR') as cm:
    ...
self.assertEqual(cm.records[0].getMessage(), ...)
```
and assert a message (`cm.records[0].getMessage()`) and an exception info (`cm.records[0].exc_info`).

Reply:
As per your request @felixxm i have added checks on `ERROR` level also.

Reply:
> As per your request @felixxm i have added checks on `ERROR` level also.

`ERROR` is still missing.

Reply:
> `ERROR` is still missing.

in the latest commit, I have added the below code in line 169:
```         
            with self.assertLogs("django.dispatch.dispatcher") as cm:
                result = a_signal.send_robust(sender=self, val="test")
            self.assertEqual(cm.records[0].getMessage(), "send_robust() received ValueError from fails")
            logErr = cm.records[0].exc_info[1]
            self.assertIsInstance(logErr, ValueError)
            self.assertEqual(logErr.args, ('this',))
```
which asserts the `ERROR` message and also checks the instance of the logError

Reply:
`ERROR` in `assertLogs()` is still missing:
```python
with self.assertLogs('django.dispatch.dispatcher', 'ERROR') as cm:
```

Reply:
Ok i got it.
Now added the 'ERROR' in assertLogs

please check the latest commit once again.